spec/requests/assessments_controller_spec.rb

```diff
@@ -101,7 +101,7 @@
     context "calling the correct workflows assessors and decorators" do
       before do
         allow(Assessment).to receive(:find).with(assessment.id.to_s).and_return(assessment)
-        allow(Workflows::MainWorkflow).to receive(:call).with(assessment).and_return(calculation_output)
+        allow(Workflows::MainWorkflow).to receive(:call).and_return(calculation_output)
```

Comment:
Its good to see the test fail if we update method params. What do you think of using .with(params) while stubbing a method with params?

Reply:
IMHO this is a terrible test, that always has to be updated,  and it has no assertions.